sdk/eventgrid/azure-messaging-eventgrid/pom.xml

```diff
@@ -12,7 +12,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-messaging-eventgrid</artifactId>
-  <version>4.7.0-beta.1</version> <!-- {x-version-update;com.azure:azure-messaging-eventgrid;current} -->
+  <version>4.7.0</version> <!-- {x-version-update;com.azure:azure-messaging-eventgrid;current} -->
```

Comment:
This has a dependency on storage-queue. So, we should release this after storage queue is released.